### DISCUSSION TITLE
Patch will not be applied

### DISCUSSION THREAD

**Discussion by vx-i** [Freetz-NG/freetz-ng#291] [Allgemein]
I would like to upgrade my AVM 1750e with Freetz-NG. Specifically, I would like to use my own certificates. There is a patch from the community for this ([link](https://www.ip-phone-forum.de/threads/wlan-repeater-1750e-modifizieren.301663/page-3#post-2307401)). I've stored the file here: patches/devices/07_0X/1750/ and build the firmware, but the patch was not applied. I would appreciate a little help or a hint.

PS: I used 7.24 labor firmware.

**Comment by fda77** [comment]
When you put the patch in 07_0X you have to select 7.0x in menuconfig

**Comment by vx-i** [comment]
I already suspected that, but I didn't find it anywhere in the docs. As I'm using 7.24 LAB I thought the folder should be named 07_24_LABOR, but it doesn't work. The key used for firmware choice in make menuconfig is 07_25_LABOR, doesn't work neither. So folder name 07_25 seems to be the right one, but it's also not working. Or the patch isn't working. I don't know.

**Reply by fda77** [reply]
Me too

**Reply by vx-i** [reply]
@fda77 This is not helpful. Who can help or how I can help myself? Any advice to source code / documentation / command?

**Reply by fda77** [reply]
First step to resolve a problem is to know what the problem is

**Reply by vx-i** [reply]
I have a patch. Now I know where to put this patch, but this one is not picked up. The problem is I don't know how to get more information.

This is the third time that I'm asking for help. I don't know what I'm supposed to have upset you with, but if you don't want to see a problem. Anyway, your answers were not helpful. I hope there are other colleagues in the Freetz-NG community who would like to give constructive advice.

Have a nice day and don't get annoyed so quickly 😉

**Reply by fda77** [reply]
Its not up to me to teach people how to ask a question or report a problem
